javascript/javascript3/week1/WeatherApp/script.js

```diff
@@ -0,0 +1,33 @@
+/////////////////////////////////////////////////////////////Weather app
+
+const but = document.getElementById('getCity');
+const cityInput = document.getElementById('cityName');
+
+//get city name and put it in a varibale by clicking the button
+
+but.addEventListener('click', selectCity);
+const load = document.getElementById('loadingBar')
+const content = document.querySelector('#content')
+const cloud = document.querySelector('#cloud')
+
+async function selectCity() {
+    const city = cityInput.value;
+    try {
+        if (city) {
+            content.innerHTML = '';
+            loader.style.display = 'block'
+            let weatherData = await fetch(`http://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=b33af74225294d0f38ecf99fd7eaca9a`)
```

Comment:
you can write "then" after fetch like:
fetch(`http://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=b33af74225294d0f38ecf99fd7eaca9a`)
. then(response => response.json())